data/java.yaml

```diff
@@ -13,7 +13,7 @@ sources:
   attributes:
     paths:
     - '%%users.appdata%%\Sun\Java\Deployment\cache\**'
-    - '%%users.userprofile%%\AppData\LocalLow\Sun\Java\Deployment\cache\**'
+    - '%%users.localappdata_low%%\Sun\Java\Deployment\cache\**'
```

Comment:
please keep this one as-is since `%localappdata_low%` does not exist as a default environment variable